packages/types/src/store/common/store.ts

```diff
@@ -1,3 +1,14 @@
+import { BaseFilterable } from "../../dal"
+
 export interface StoreDTO {
   id: string
+  name: string
+  default_sales_channel_id?: string
+  default_region_id?: string
+  metadata?: Record<string, any>
```

Comment:
todo: rm `?` and add `| null`

Reply:
@adrien2p that's fine with me, but why do we prefer null over an optional?

Reply:
Because it will be returned but with `null` as the default value is null no? the only moment it wont be returned will be if you don't select it. But this will be part of a dynamic typing i ll do at a later point.

Reply:
and the foreign keys too normally